app/validators/value_source_validator.py

```diff
@@ -49,6 +53,13 @@ def _validate_list_source_reference(self, identifiers):
             if identifier not in self.questionnaire_schema.list_names:
                 self.add_error(self.LIST_REFERENCE_INVALID, identifier=identifier)
 
+    def _validate_response_metadata_source_reference(self, identifiers):
+        for identifier in identifiers:
+            if identifier not in ["started_at"]:
```

Comment:
This looks a bit odd, my feeling is this has been done as you see response_metadata_source expanding, which is fine, however if that was the case I would expect it to be a constant REPSONSE_METADATA_IDENITIFERS. On the other hand I wonder if we have any other use case above started_at, if not then this should be a straight equals and extend when needed

Reply:
At the moment only `started_at` as far as I know, should I make it that one identifier only without a list or create a constant with one element for now?